src/classes/ExtendedGraph.class.php

```diff
@@ -549,7 +549,10 @@ public function from_rdfxml($rdfxml, $base='') {
     public function from_json($json) {
         if ($json) {
             $this->remove_all_triples();
-            $this->_index = json_decode($json, true);
+            $index = json_decode($json, true);
+            if (is_array($index)) {
```

Comment:
I was going to ask why we need to check if its an array, then I remembered that `$this->_index` is initalised as an empty array, and json_decode here will return either an associative array or `null`. I'm guessing as it was it was generating warnings.